lib/brain.js

```diff
@@ -2041,13 +2041,16 @@ Brain.prototype.smsCompliance = function smsCompliance (opts = {}) {
   flow.handle('start')
 }
 
-Brain.prototype.validateTier = function validateTier (tier, triggerTx, customerTier) {
+Brain.prototype.validateTier = function validateTier (tier, triggerTx, customerTier, customer) {
   const requiredTiers = this.requiredTiers(this.trader.triggers, this.customerTxHistory, triggerTx)
   const requiredManualTiers = _.filter(tier => _.get(tier)(this.trader.triggersAutomation) === MANUAL)(requiredTiers)
   const scannedManualTiers = _.filter(tier => _.get(tier)(this.manualTriggersDataProvided))(requiredManualTiers)
 
+  // If the tier is not manual, allow pending values
   if (!_.includes(tier, requiredManualTiers)) return !_.isEmpty(customerTier)
-  this.txBlockedByManualTrigger = true
+
+  // If the tier is manual, block pending
+  if (!_.isEmpty(customer)) this.txBlockedByManualTrigger = true
```

Comment:
Why are we testing for customer before this? Won't this skip blocking the tx if a trigger is set as manual to a higher threshold? 

Why even try to pass down customer if you already have the customerTier?